### DISCUSSION TITLE
How to set up a repo card?

### DISCUSSION THREAD

**Discussion by snowy-jaguar** [anuraghazra/github-readme-stats#1682] [Q&A] [answered]
I'm looking to make a card that shows the stats of a specific repo but unless I'm blind I'm not seeing an example on how to access the repo card endpoint.

**Comment by rickstaa** [comment, marked as answer]
@SnowyJaguar1034 If your mean the Repo card see https://github.com/anuraghazra/github-readme-stats#github-extra-pins. 

If you however want to show the stats and language card results for only one repository this has not yet been implemented. There is a pull request open to adding this feature to the language card (i.e. 1590). I think adding this to the stats card does not make sense this card is meant to show the users' stats, not the repo stats. If you want to show an overview of the number of watchers, forks and stars for a repository a new card should be added for that.
